wolox-books/src/app/screens/shopping-cart/components/list/list.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { AppState } from 'src/app/app.state';
+import * as ShoppingCartActions from '../../../../store/shopping-cart.action';
+import { Book } from 'src/app/screens/book/book';
+
+@Component({
+  selector: 'shopping-cart-list',
+  templateUrl: './list.component.html',
+  styleUrls: ['./list.component.scss']
+})
+export class ListComponent {
+
+  books: Book[];
+
+  _open: boolean;
```

Comment:
why this?

Reply:
acts like a two way binding state, `_open` sets the initial state on constructor that the get won't be pass if I don't get the value from the observable that must be more complex the methods.